starter-code/css/main.css

```diff
@@ -6,3 +6,158 @@ Green: #00B172
 White: #FFF
 
 */
+
+* {
+    font-family: 'Raleway', sans-serif;
+}
+
+.nav {
+    background-color: white;
+    width: 100vw;
+    height: 60px;
+    top: 12px;
+    position: fixed;
+}
+.nav img {
+    margin-top: -4px;
+    height: 50px;
+    width: 130px;
+    margin-left: 30px;
+}
+
+.nav ul {
+    margin-top: 10px;
+    margin-right: 25px;
+    display: flex;
+    float: right;
+    list-style-type: none;
+}
+
+.nav li {
+    padding-right: 20px;
+}
+
+.top-section {
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-image: url('/Users/Drewskii/Desktop/labs/code/lab-css-spotify-clone/starter-code/images/landing.jpg')
+}
+
+.top-section h1 {
+    position: relative;
+    text-align: center;
+    display: block;
+    top: 220px;
+    font-size: 50px;
+    color: white;
+    height: 700px;
+}
+
+
+.text {
+   position: relative;
+   text-align: center;
+    top: -420px;
+    font-size: 23px;
+    color: white;
+    width: 26%;
+    margin: 0 auto;
+}
+
+.whats-on {
+    text-align: center;
+    font-size: 20px;
+}
+
+hr {
+    text-align: center;
+    width: 17%;
+    border-top: solid 2px;
+    color: #00B172;
+    margin-top: -13px;
+}
+
+.middle-body img {
+    height: 85px;
+    width: 85px;
+}
+
+.group {
+    display: flex;
+    flex-direction: row;
+    margin-top: 70px;
+    justify-content: center;
+}
+
+.million, .hd, .stream {
+    display: flex;
+    flex-direction: column;
+    padding-right: 50px;
```

Comment:
a little more padding on both sides. about 130px would have been good for the spacing. Also another way to control the padding on both sides is like this `padding: 0 130px 0 130px`. the first number is for the top padding then the right, bottom and finally left.  It goes clockwise.